client.py

```diff
@@ -82,4 +85,55 @@ async def roll(ctx: SlashContext, num_sides: int = 6, num_dice: int = 2):
   await ctx.send(content=f"Rolled {num_dice} {num_sides}-sided dice. Output: {rolls}, Total: {sum(rolls)}")
 
 
-bot.run(BOT_TOKEN)
+@slash.slash(
+    name="save",
+    guild_ids=GUILD_IDS,
+    description="Update a key/value",
+    options=[
+        create_option(
+            name="key",
+            description="key",
+            option_type=SlashCommandOptionType.STRING,
+            required=True,
+        ),
+        create_option(
+            name="value",
+            description="value",
+            option_type=SlashCommandOptionType.STRING,
+            required=True,
+        ),
+    ])
+async def save(ctx: SlashContext, key, value):
+    try:
+        decoded = json.loads(value)
+    except JSONDecodeError:
+        decoded = value
+    ds.data[key] = decoded
+    formatted = json.dumps(ds.data, indent=2)
+    embed = Embed(title="Datastore", color=Color.blurple())
+    embed.add_field(name="result", value=f"```json\n{formatted}\n```", inline=False)
+    await ctx.send(embed=embed)

```

Comment:
Depending on the expected size of the json, it might be better to only display the added field instead of everything? Maybe have ellipses instead.

On that note, maybe a delete and "see whole json" might be nice (if length expected to be long maybe give it a file to download instead of embed).